derive/src/rpc_attr.rs

```diff
@@ -162,24 +168,28 @@ fn validate_attribute_meta(meta: syn::Meta) -> Result<syn::Meta> {
 	}
 	impl<'a> Visit<'a> for Visitor {
 		fn visit_meta(&mut self, meta: &syn::Meta) {
+			let ident = meta.path().get_ident().map(|i| i.to_string())
+				// TODO [ToDr] Maybe better fail?
+				.unwrap_or_else(|| "Unknown".into());
 			match meta {
-				syn::Meta::List(list) => self.meta_list_names.push(list.ident.to_string()),
-				syn::Meta::Word(ident) => self.meta_words.push(ident.to_string()),
-				syn::Meta::NameValue(nv) => self.name_value_names.push(nv.ident.to_string()),
+				syn::Meta::Path(_) => self.meta_words.push(ident),
+				syn::Meta::List(_) => self.meta_list_names.push(ident),
+				syn::Meta::NameValue(_) => self.name_value_names.push(ident),
 			}
 		}
 	}
 
 	let mut visitor = Visitor::default();
 	visit::visit_meta(&mut visitor, &meta);
 
-	match meta.name().to_string().as_ref() {
-		RPC_ATTR_NAME => {
+	let ident = meta.path().get_ident().map(|i| i.to_string());
```

Comment:
would almost make sense the extract this pattern to separate function, `fn meta_into_str<'a>() -> Option<&'a str>` because it used a couple of times